solution-base/deps.yaml

```diff
@@ -3,10 +3,10 @@
 # yq eval 'sortKeys(.)' -i deps.yaml
 mongodb-sharded:
   image: bitnami/mongodb-sharded
-  tag: 7.0.6-debian-12-r0
+  tag: 8.0.10-debian-12-r2
 mongodb-sharded-exporter:
   image: bitnami/mongodb-exporter
-  tag: 0.40.0-debian-12-r12
+  tag: 0.45.0-debian-12-r3
 mongodb-shell:
   image: bitnami/os-shell
-  tag: 12-debian-12-r22
+  tag: 12-debian-12-r46
```

Comment:
In the chart we use different versions :
- bitnami/mongodb-sharded:8.0.12-debian-12-r0
- bitnami/mongodb-exporter:0.45.0-debian-12-r5
- bitnami/os-shell:12-debian-12-r49

Reply:
there is no problem with having different versions "in the chart" : whatever "default" is in the chart, we want to specify here what we use...
...however we should indeed use the latest & greatest patch releases